app/components/object-inspector/property.js

```diff
@@ -54,10 +88,13 @@ export default Component.extend({
   },
 
   cannotEdit() {
+    if (this.model.name === '...') return true;
+    if (!this.isCalculated) return true;
```

Comment:
Could we combine these lines?
```suggestion
   if (this.model.name === '...' || !this.isCalculated) return true;
```